src/database/models/ItemModel.ts

```diff
@@ -0,0 +1,40 @@
+import { DataTypes, Model } from 'sequelize'
+import sequelizeConnection from '../config/db.config'
+
+interface ItemAttributes {
+  id?: number
+  name: string
+  description: string
+}
+
+class Item extends Model<ItemAttributes> implements ItemAttributes {
+  public id!: number
+  public name!: string
+  public description!: string
+}
+
+Item.init(
+  {
+    id: {
+      type: DataTypes.INTEGER,
+      autoIncrement: true,
+      primaryKey: true,
+    },
+    name: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    description: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
```

Comment:
You can add other attributes like price, category, Images, expireDate,... and others